pdfminer/pdfpage.py

```diff
@@ -110,11 +115,18 @@ def search(
             elif tree_type is LITERAL_PAGE:
                 log.info('Page: %r', tree)
                 yield (objid, tree)
+
+        try:
+            page_labels: Optional[Iterator[str]] = document.get_page_labels()
+        except PDFNoPageLabels:
+            page_labels = None
```

Comment:
Maybe use `itertools.count()` here

Reply:
Not sure what you meant about `count`, but I did manage to clean this up quite nicely with `itertools.repeat`.